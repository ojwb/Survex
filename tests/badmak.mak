/ pos=fail warn=1 error=3 /
#backread.dat,
C1 [inch,10.23,20.47,1234.56], C3 [m,10.23,20.47,1234.56],
C2 [n,10.23,20.47,1234.56], C3 [m,10.23,20.47,1234.56],
C3 [f,10.23,20.47,1234.56];
