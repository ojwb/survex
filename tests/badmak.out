./badmak.mak:3:5: error: Expecting "F" or "M"
 C1 [inch,10.23,20.47,1234.56], C3 [m,10.23,20.47,1234.56],
     ^
./badmak.mak:4:5: error: Expecting "F" or "M"
 C2 [n,10.23,20.47,1234.56], C3 [m,10.23,20.47,1234.56],
     ^
./badmak.mak:4:29: warning: Station already fixed at the same coordinates
 C2 [n,10.23,20.47,1234.56], C3 [m,10.23,20.47,1234.56],
                             ^~
./badmak.mak:3:32: info: Previously fixed or equated here
./badmak.mak:5:1: error: Station already fixed or equated to a fixed point
 C3 [f,10.23,20.47,1234.56];
 ^~
./badmak.mak:3:32: info: Previously fixed or equated here
./badmak.mak:15:1: warning: Unknown command "?what;"
 ?what;
 ^

Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Calculating statistics...

Survey contains 6 survey stations, joined by 5 legs.
There are 0 loops.
Total length of survey legs =    5.39m (   3.39m adjusted)
Total plan length of survey legs =    5.29m
Total vertical length of survey legs =    0.56m
Vertical range = 0.52m (from D1 at 1235.08m to B2 at 1234.56m)
North-South range = 2.15m (from D1 at 21.62m to B2 at 19.47m)
East-West range = 1.00m (from B2 at 11.23m to C3 at 10.23m)

There were 2 warning(s) and 3 error(s) - no output files produced.
